Pollo/Models/ListDiffableModels/Draft.swift

```diff
@@ -15,12 +15,13 @@ class Draft: Codable {
     var id: String
     var options: [String]
     var text: String
+    var isLoaded: Bool? = false
```

Comment:
Why is this an optional?

Reply:
Because `Draft` conforms to `Codable`, I needed this field without changing the model that the backend uses, therefore `isLoaded` being optional is the solution.